indigo/settings.py

```diff
@@ -149,7 +149,7 @@
 
     # AKN document types the platform supports, as a list of (name, code) tuples
     # see http://docs.oasis-open.org/legaldocml/akn-core/v1.0/os/part1-vocabulary/akn-core-v1.0-os-part1-vocabulary.html#_Toc523925025
-    'DOCTYPES': [('Act', 'act')],
+    'DOCTYPES': [('Act', 'act'), ('Statement', 'statement')],
```

Comment:
otherwise the tests fail since we expect to be able to create statements. @goose-life I don't know if you want us to keep this in, or if we should adjust the tests for this test case.